src/julia/darray.jl

```diff
@@ -128,6 +161,21 @@ function foxLi{T<:BlasComplex}(::Type{T}, n::Integer, ω::Real)
 end
 foxLi(n::Integer, ω::Real) = foxLi(Complex128, n, ω)
 
+function uniform{T<:BlasFloat}(::Type{T}, m::Integer, n::Integer)
```

Comment:
You have a point here, but it is really something that should be handled within Julia's existing framework for random number and DArrays. I think we a doing okay, i.e. using different seed for our Mersenne-Twister on each worker.

What are you doing in Elemental to ensure independece?